swagger.yaml

```diff
@@ -6244,7 +6304,6 @@ definitions:
         type: string
         format: uuid
         description: Unique identifier of the cluster this event relates to.
-        x-go-custom-tag: gorm:"index"
```

Comment:
While the cluster_id isn't mandatory, the index itself might be useful in this context when searching for events based on cluster-id.
I'd explore the need to add indices also for infra-env-id/host-id which the API allows to search by.

Reply:
how will this behave for events with no cluster id?

Reply:
you can index a column which allows null values

Reply:
lgtm